src/Dropdown/Dropdown.tsx

```diff
@@ -9,8 +9,13 @@ import { makeDefaultCSS } from './helpers'
 import { fromTheme } from '../utils/styled'
 import { useSnapEdgeToParent, Vertical, Horizontal, BreakDistance } from '../useSnapEdgeToParent'
 
-const StyledDropdown = styled(Block)`
+interface StyledDropdownProps {
+  open: boolean
+}
+
+const StyledDropdown = styled(Block) <StyledDropdownProps>`
   position: relative;
+  ${({ open }) => !open && css`overflow: hidden;`}
```

Comment:
@Stefanwullems do you remember why this was added? It causes minor glitches in certain conditions:

![image](https://user-images.githubusercontent.com/205834/64433046-02bfd200-d0be-11e9-87dc-3c108fb9696e.png)